### PR TITLE
small fix based on pullapprove document

### PR DESCRIPTION
**Pull request by Mashimiao**
Other projects may refer to template's pullapprove config file.
I think we'd better based on pullapprove's document.
This will reduce doubts of pullapprove grammar.

Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -4,19 +4,20 @@ requirements:
   signed_off_by:
     required: true
 
+always_pending:
+  title_regex: '^WIP'
+    explanation: 'Work in progress...'
```
Comment (wking) [inline .pullapprove.yml]: This line needs a two-space indent (not a four-space indent) for this to be [valid YAML][1].

[1]: https://pullapprove.com/opencontainers/project-template/pull-request/38/

Comment (Mashimiao) [inline .pullapprove.yml]: Oh!!!  Thanks, updated. 
But it's still very strange. `selinux-maintainers` really exists on opencontaienrs repo, but pullapprove says not.

Comment (wking) [inline .pullapprove.yml]: > `selinux-maintainers` really exists on opencontaienrs repo, but pullapprove says not.

PullApprove [says][1]:

> selinux-maintainers team not found on this repo

And looking at our selinux repo, it looks like the group is `go-selinux-maintainers` (e.g. see [here][2] and [here][3]).  I've filed #39 to catch up with the current group name, but the org admins may want to rename the group to match the current repo name.

[1]: https://pullapprove.com/opencontainers/project-template/pull-request/38/requirements/
[2]: https://github.com/opencontainers/selinux/blob/4a2974bf1ee960774ffd517717f1f45325af0206/.pullapprove.yml#L9
[3]: https://pullapprove.com/opencontainers/go-selinux/pull-request/15/reviews/

Comment (Mashimiao) [inline .pullapprove.yml]: That's why I said very strange. At https://github.com/orgs/opencontainers/teams/, we can say `selinux-maintainers` exists. `go-selinux-maintainers` does not exist.

**Pull request merged**